src/reports/components/report-sections/full-rule-header.tsx

```diff
@@ -44,21 +43,29 @@ export const FullRuleHeader = NamedFC<FullRuleHeaderProps>('FullRuleHeader', pro
     const renderRuleLink = () => {
         const ruleId = cardResult.id;
         const ruleUrl = cardResult.url;
-        return (
-            <span className="rule-details-id">
-                <NewTabLink
-                    href={ruleUrl}
-                    aria-label={`rule ${ruleId}`}
-                    aria-describedby={ariaDescribedBy}
-                >
-                    {ruleId}
-                </NewTabLink>
-            </span>
+        const displayedRule = ruleUrl ? (
+            <NewTabLink
+                href={ruleUrl}
+                aria-label={`rule ${ruleId}`}
```

Comment:
I see that this PR did not introduce the aria-attributes here (they were already there), but I am curious about why they differ from other places we render the rule links (like [here](https://github.com/microsoft/accessibility-insights-web/blob/3459929849613cd7403afd8b568594a99a950361/src/common/components/cards/rule-resources.tsx#L38)). Is one approach better to follow?